resources/lang/en/chat.php

```diff
@@ -13,18 +14,43 @@
         'user' => 'You cannot message this user at this time.',
     ],
 
+    'channels' => [
+        'join' => 'join channel',
```

Comment:
as nothing seem to be set in stone yet regarding all this join channel and create channel/group/announcement thing, this should say new announcement for now (or something in that effect). Especially if there will be different interface for each action